presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for Joni regular expression functions.
+* Fix a correctness bug that would occur if two tables with mismatching but compatible bucket counts are joined whiles filters are applied to one of the tables.
+* Fix Parquet predicate pushdown for ``smallint`` and ``tinyint`` types.
+* Fix a failure that occurs when view columns were fully qualified with catalog and/or schema.
+* Fix a failure that occurs then creating and reading an unpartitioned Hive table within the same transaction.
+* Fix an issue that causes procedure calls to fail if no queries have been run after the server startup.
+* Fix an issue in analyzing property system table. If a connector is removed, its analyze properties will also be removed in the ``system.metadata.analyze_properties`` table.
+* Add :func:`ST_Length` for spherical geography
+* Add ``view_owner`` column to the ``information_schema.views`` system table.
+* Add a json version of the query plan to ``QueryCompletedEvent``.
+* Add support for creating warnings during parsing.
+* Add a warning for using reserved word ``current_role`` as an identifier.
+* Add support for using the empty string as a delimiter for the :func:`split` function.
+  When an empty string is used as the delimiter, the string will be split into individual characters.
+
+Raptor Changes
+--------------
+* Add ``raptor.minimum-node-count`` configuration property. The the number of nodes in a cluster is fewer then the configured amount, data recovery and reassignment will not run.
+* Advance the first run of bucket rebalancer. Now bucket balancer will run after a grace period (configured in `raptor.startup-grace-period`) from coordinator restart.
+
+Hive Connector Changes
+----------------------
+
+* Add `hive.ignore-table-bucketing` configuration property and `ignore_table_bucketing` session property.  When set to true these properties enable reading from partitions whose bucketing scheme does not match the table bucketing scheme.
+
+Verifier Changes
+----------------
+* Fix an issue for ``Insert`` query verification where determinism check is not ran in case of result mismatch.
```

Comment:
Fix an issue where the determinism check for ``Insert`` queries was not run when there was a result mismatch.